src/fdcache_entity.cpp

```diff
@@ -526,7 +526,9 @@ int FdEntity::Open(const headers_t* pmeta, off_t size, const struct timespec& ts
                         need_save_csf = true;     // need to update page info
                     }
                     if(size != st.st_size){
+                        pagelist.Init(size, false, false);
```

Comment:
I don't think there is any need to modify the pagelist here.
If it is necessary, it is judged by the following judgment statement (processing several lines before), and correction has already been made.
```
L524:    if(size != pagelist.Size()){
```